templates/pytorch-1.1.x/src/__ALGO___test.py

```diff
@@ -0,0 +1,8 @@
+from __ALGO__ import apply
+
+def test_algorithm():
+    input = {"matrix_a": [[0, 1], [1, 0]], "matrix_b": [[25, 25], [11, 11]]}
+    result = apply(input)
+    assert result == {"product": [[11., 11.], [25., 25.]]}
```

Comment:
do we really need the dots? 11 and `11.` are going to be equal always, right/

Reply:
I think the results are floats, hence why we need the dots.

Reply:
Also, I think we want to be explicit about it.